File: wallet/src/main/java/com/worldventures/wallet/analytics/firmware/action/ViewSdkVersionAction.java
package com.worldventures.wallet.analytics.firmware.action;

import com.worldventures.core.service.analytics.AdobeTracker;
import com.worldventures.janet.analytics.AnalyticsEvent;

@AnalyticsEvent(action = "wallet:settings:general:SmartCard Update:SmartCard Up to Date",
                navigationState = true,
                trackers = AdobeTracker.TRACKER_KEY)
public class ViewSdkVersionAction extends FirmwareAnalyticsAction {
}

File: wallet/src/main/java/com/worldventures/wallet/analytics/firmware/action/ViewSdkVersionAction.kt
package com.worldventures.wallet.analytics.firmware.action

import com.worldventures.janet.analytics.AnalyticsEvent
import com.worldventures.core.service.analytics.AdobeTracker

@AnalyticsEvent(action = "wallet:settings:general:SmartCard Update:SmartCard Up to Date",
      navigationState = true, trackers = arrayOf(AdobeTracker.TRACKER_KEY))
class ViewSdkVersionAction : FirmwareAnalyticsAction()
